Library/Formula/kitchen-sync.rb

```diff
@@ -0,0 +1,23 @@
+class KitchenSync < Formula
+  desc "Fast efficiently sync database without dumping & reloading"
+  homepage "https://github.com/willbryant/kitchen_sync"
+  url "https://github.com/willbryant/kitchen_sync/archive/0.37.tar.gz"
+  sha256 "a78206cdd3ac4d4aa646cdfc8ae34c5df87179223c59d5dcca625760c68ae4e7"
+  head "https://github.com/willbryant/kitchen_sync.git"
+
+  depends_on "cmake" => :build
+  depends_on "boost"
+  depends_on :mysql => :recommended
+  depends_on :postgresql => :optional
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    assert File.exist?("#{bin}/ks")
+    assert File.exist?("#{bin}/ks_mysql") if build.with? "mysql"
+    assert File.exist?("#{bin}/ks_postgresql") if build.with? "postgresql"
```

Comment:
Hey guys, great to see this.  I believe you can remove the openssl dependency, that was my bad - I forgot to update the installation instructions when I added direct support for Common Crypto on OS X, so now OpenSSL is only needed on Linux and so on.

Let me know if it isn't compiling without it or something - unfortunately all my machines have OpenSSL already so hard for me to test this properly.  All I know for sure is that we don't get the deprecation warnings...